### PR TITLE
goimports

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

**Comment by codycushing** [comment]
Tested and works, did require cleaning and updating my x/tools/* source. To reduce unnecessary merge conflicts, we should do this change with Ramys rename task, I'll make sure it gets included there.

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ clean:
 	rm -rf bin/*
 
 fmt:
-	gofmt -w $(GOFMT_FILES)
+	goimports -w -local github.com/oracle/terraform-provider-baremetal $(GOFMT_FILES)
```
Comment (codycushing) [inline Makefile]: @rcohenma can you include this change in your current rename branch then run `make fmt`.

**Comment by josephholsten** [comment]
I was just going to ask if anyone had any significant branches in progress ;-). I'll rip out the actual fmt change so we can apply it when things are less in flux.

**Comment by josephholsten** [comment]
Updated. Should be boring to merge, and we can fmt when it's convenient.

**Pull request merged**